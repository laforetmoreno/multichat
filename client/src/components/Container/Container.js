import React from 'react';
import { node, bool, string } from 'prop-types';

import styles from './Container.module.scss';

const Container = ({ children, flex, direction, align }) => (
  <main
    className={`
    ${styles.wrapper}
    ${flex && styles.flex}
    ${direction && styles[direction]}
    ${align && styles[align]}`}
  >
    {children}
  </main>
);

Container.propTypes = {
  children: node.isRequired,
  flex: bool,
  direction: string,
};

export default Container;
