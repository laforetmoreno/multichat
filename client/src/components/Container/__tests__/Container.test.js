import React from 'react';
import Container from '../Container';
import renderer from 'react-test-renderer';
import { unmountComponentAtNode } from "react-dom";

describe('<Container />', () => {
  it('renders correctly', () => {
    const tree = renderer
      .create(
        <Container>
          <h1>Hello</h1>
        </Container>
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});