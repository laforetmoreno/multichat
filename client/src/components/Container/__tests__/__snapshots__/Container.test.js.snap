// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Container /> renders correctly 1`] = `
<main
  className="
    wrapper
    undefined
    undefined
    undefined
    undefined"
>
  <h1>
    Hello
  </h1>
</main>
`;
